test-jest/cartesian/Brush.spec.tsx

```diff
@@ -95,7 +103,11 @@ describe('<Brush />', () => {
   });
 
   test('render text when alwaysShowText is true', () => {
-    const { container } = render(<Brush x={100} y={50} width={400} height={40} data={data} alwaysShowText />);
+    const { container } = render(
+      <BarChart width={500} height={100} data={data}>
```

Comment:
any reason this is wrapped in a bar chart?

Reply:
Same as this [comment](https://github.com/recharts/recharts/pull/3269#discussion_r1084847142)